servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/ServicetalkNetServerAttributesGetter.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2023 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicetalk.opentelemetry.http;
+
+import io.servicetalk.http.api.HttpRequestMetaData;
+import io.servicetalk.http.api.HttpResponseMetaData;
+import io.servicetalk.transport.api.HostAndPort;
+
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetServerAttributesGetter;
+
+import javax.annotation.Nullable;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+final class ServicetalkNetServerAttributesGetter
```

Comment:
Merged into a single `ServicetalkNetAttributesGetter` class to reduce duplication for common methods